doc/src/manual/parallel-computing.md

```diff
@@ -1346,288 +1639,158 @@ Currently, sending a message between unconnected workers results in an error. Th
-provides this functionality. Yield points provide for co-operatively scheduling multiple tasks
-onto the same OS thread. I/O tasks and timers yield implicitly while waiting for the event to
-occur. Calling [`yield`](@ref) explicitly allows for other tasks to be scheduled.
+comm = MPI.COMM_WORLD
+MPI.Barrier(comm)
 
-Thus, a task executing a [`ccall`](@ref) effectively prevents the Julia scheduler from executing any other
-tasks till the call returns. This is true for all calls into external libraries. Exceptions are
-calls into custom C code that call back into Julia (which may then yield) or C code that calls
-`jl_yield()` (C equivalent of [`yield`](@ref)).
+root = 0
+r = MPI.Comm_rank(comm)
 
-Note that while Julia code runs on a single thread (by default), libraries used by Julia may launch
-their own internal threads. For example, the BLAS library may start as many threads as there are
-cores on a machine.
+sr = MPI.Reduce(r, MPI.SUM, root, comm)
 
-The [`@threadcall`](@ref) macro addresses scenarios where we do not want a [`ccall`](@ref) to block the main Julia
-event loop. It schedules a C function for execution in a separate thread. A threadpool with a
-default size of 4 is used for this. The size of the threadpool is controlled via environment variable
-`UV_THREADPOOL_SIZE`. While waiting for a free thread, and during function execution once a thread
-is available, the requesting task (on the main Julia event loop) yields to other tasks. Note that
-`@threadcall` does not return till the execution is complete. From a user point of view, it is
-therefore a blocking call like other Julia APIs.
+if(MPI.Comm_rank(comm) == root)
+   @printf("sum of ranks: %s\n", sr)
+end
 
-It is very important that the called function does not call back into Julia.
+MPI.Finalize()
+```
 
-`@threadcall` may be removed/changed in future versions of Julia.
+```
+mpirun -np 4 ./julia example.jl
+```
 
 [^1]:
-    In this context, MPI refers to the MPI-1 standard. Beginning with MPI-2, the MPI standards committee
-    introduced a new set of communication mechanisms, collectively referred to as Remote Memory Access
-    (RMA). The motivation for adding RMA to the MPI standard was to facilitate one-sided communication
-    patterns. For additional information on the latest MPI standard, see [http://mpi-forum.org/docs](http://mpi-forum.org/docs/).
+    in this context, mpi refers to the mpi-1 standard. beginning with mpi-2, the mpi standards committee
+    introduced a new set of communication mechanisms, collectively referred to as remote memory access
+    (rma). the motivation for adding rma to the mpi standard was to facilitate one-sided communication
+    patterns. for additional information on the latest mpi standard, see [http://mpi-forum.org/docs](http://mpi-forum.org/docs/).
+ 
+ [^2]:
+    [Julia GPU man pages](http://juliagpu.github.io/CUDAnative.jl/stable/man/usage.html#Julia-support-1) cc: @maleadt
```

Comment:
pretty sure you didn't intend to add that `cc:` there :-)

Reply:
Sorry changed! I don't know if how to mention properly in Markdown, please let me know!